### DISCUSSION TITLE
Question 2: how to select model

### DISCUSSION THREAD

**Discussion by mybirth0407** [Boostcamp-AI-Tech-1-15/ddobokki_log#30] [2. DL Basic]
정확도가 85인 설명가능한 모델과, 정확도가 95인 설명불가능한 모델이 있을 때, 어떤 모델을 더 선호하고 그 이유가 무엇인가요?

**Comment by ddeokbboki-good** [comment]
95인 모델
차차 알아가면 된다, bathcnormalization도 완벽하게 설명하기 어려운데 많이 사용하는 것처럼 정확도 높은 것을 사용하겠다.

**Reply by ddobokki** [reply]
케바케인것 같습니다.

프레젠테이션이 목적인 경우는 설명 가능한 모델을 선택해야 할 것이고
실제 서비스에서 사용하여 사용자는 모델의 결과만이 중요할 때는 성능좋은 설명 불가능한 모델을 선택할 것 같습니다.

**Comment by minji-o-j** [comment]
분야에 따라서 사용하는 모델이 다를 것 같지만, (단순 서비스 or 추천시스템: 95%, 인사 판단 등 책임이 필요한 결정을 내릴 때 등: 85%모델..)

저는 85% 모델을 선호합니다. 
왜냐하면 설명이 가능하다면, 15%를 틀리는 이유도 찾아내어 개선이 가능할 것이라고 생각합니다.
또한 설명 가능한 모델을 이용해서 새로운 데이터 패턴을 찾을 수 있기 때문에 이를 이용한 새로운 서비스를 만드는 데에도 기여할 수 있다고 생각하기 때문에 설명 가능한 모델을 더 선호합니다.

~~마지막말은 최성준교수님이 내리신 무엇이 인공지능인가? 에 대한 답변이 생각나서..ㅎㅎ~~

**Comment by mybirth0407** [comment]
좋습니다. 이건 답이 있기보다는 개인의 의견/성향 차이인데요.
보통 제 경험상 85% 를 선호하시는 분들은 datascientist, analyst의 성향에 가깝고, 95% 를 선호하시는 분들은 engineer 성향에 가깝다고 생각이 들어요!
이유는, 해석을 통해 변화/개선할 수 있는 부분들을 가지고 집단에 기여하는 역할이 전자이고,
빠르게 모델을 구현/배포해서 잘 서비스할 수 있도록 하는 역할을 후자가 많이 하게 되서 그런 듯 합니다!
개인 진로와도 한번 연결지어보면 좋을 듯 해요! (물론 제 개인적인 의견이고, 절대적이진 않습니다ㅋㅋ)
